src/clustering/worker/Worker.ts

```diff
@@ -114,7 +116,7 @@ export class Worker extends Emitter<DiscordEventMap> {
     return this.api instanceof RestManager && this.shards.every(x => x.ready)
   }
 
-  log (data: string) {
-    this.comms.tell('LOG', data)
+  log (...data: any) {
+    this.comms.tell('LOG', data.map(d => typeof d === 'string' ? d : inspect(d)).join(' '))
```

Comment:
You need to change the cluster comms as well. Change the `ThreadEvents` to allow for a `string[]` or `string`

It might also be beneficial to scrap multi-message entirely as it's not possible to pass multiple strings to the `Master.log()` as the second parameter is a `Cluster` object.
Just add the inspect on the log methods and only take on string.

Also for further development I recommend just using console.debug when you need to log out an entire object that needs string inspecting.

Reply:
Actually yeah, only use one string please.